force-app/main/default/staticresources/pmm/recentSessionsOverrides.css

```diff
@@ -7,15 +7,15 @@
  *
  */
 
-.slds-card__body {
+.sessions .slds-card__body {
```

Comment:
Just one small question. Do we have to add the class to the static resource or could we have added .sessions class definition to the css file that is within the component